packages/node-api/integration/LoAuthorityList.ts

```diff
@@ -50,6 +53,7 @@ export async function getAvailableRegions() {
 
 export async function importHost() {
     const { api, alice } = await setup();
+    const ferdie = encodeAddress("5CiPPseXPECbkjWCa6MnjNokrgYjMqmKndv2rSnekmSK2DjL", api.queries.ss58Prefix);
```

Comment:
Could use `getValidAccountId` if applying my previous comment.

Reply:
Done in b8dee278